oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/IndexerStatisticsTracker.java

```diff
@@ -77,7 +77,7 @@ public void onEntryEnd(String entryPath) {
     public String formatStats() {
         long endTimeNanos = System.nanoTime();
         long totalTimeNanos = endTimeNanos - startIndexingNanos;
-        double avgTimePerDocumentMicros = FormattingUtils.safeComputeAverage(totalIndexingTimeNanos/1000, nodesIndexed);
+        long avgTimePerDocumentMicros = Math.round(FormattingUtils.safeComputeAverage(totalIndexingTimeNanos/1000, nodesIndexed));
```

Comment:
NIT: formatting. 

Hmm I think we would benefit from a formatter. Not sure I caught all of them.

Reply:
fixed